sdk/rust/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "dstack-rust"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+ethers = "2.0"
```

ethers was deprecated over a year ago: https://github.com/gakonst/ethers-rs/issues/2667
```suggestion
alloy = { version = "0.15", features = ["signers"] }
```